### PR TITLE
Update gradle build files

### PR DESCRIPTION
**Pull request by grafixeyehero**
updating build version

### CODE REVIEW

**Comment by JustAMan** [comment]
Does this also include #31 (seeing as you've closed that one without it being merged)?

**Comment by grafixeyehero** [comment]
yes

**Comment by thornbill** [comment]
I had some issues building this. @dkanada were you able to build with these changes?

File: app/build.gradle [hunk 1]
```diff
@@ -21,30 +21,48 @@ android {
             minifyEnabled false
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
+        debug {
+            applicationIdSuffix ".debug"
+            debuggable true
+        }
+    }
+	splits {
```
Comment (dkanada) [inline app/build.gradle]:
```suggestion
    splits {
```

**Comment by thornbill** [comment]
Thanks for your contribution @grafixeyehero! @dkanada made some additional tweaks based on your changes in https://github.com/jellyfin/jellyfin-androidtv/pull/33. We wanted to make sure your contributions here were recognized, so I added you to CONTRIBUTORS.md in that PR and will be closing this PR.

**Pull request closed without merging**